<?xml version="1.0"?>

<launch>


<!-- SELECT ONE OF THE TWO NODES BELOW -->



<!--Run the node that will generate an example trajectory-->
<!--Example curves-->
<!--node pkg = "espeleo_planning" name = "trajectory_planner" type = "example_trajectories.py" args=" 1 500" output="screen">
	<rosparam command="load" file="$(find espeleo_planning)/config/planning_params.yaml" />
</node-->


<!--Run the node that will read a trajectory from the file and publish to the controller-->
<!--Curves generated by the Dijkstra algorithm-->
<node pkg = "espeleo_planning" name = "trajectory_planner" type = "dijkstra_trajectories.py" args="1" output="screen">
	<rosparam command="load" file="$(find espeleo_planning)/config/planning_params.yaml" />
</node>



</launch>
